tailwind.config.ts

```
@@ -121,7 +121,7 @@ const config: Config = {
             },
         },
     },
-    plugins: [],
+    plugins: [require("tailwind-scrollbar")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)